# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmic-comp/hotfix-match.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cosmic-comp-epoch-1.0.0-alpha.5.1/src/input/actions.rs.vanilla	2025-01-24 15:37:12.323333340 +0100
+++ cosmic-comp-epoch-1.0.0-alpha.5.1/src/input/actions.rs	2025-01-24 15:39:55.220000062 +0100
@@ -1028,6 +1028,8 @@
 
             // Do nothing
             Action::Disable => (),
+            
+            cosmic_settings_config::Action::ZoomIn | cosmic_settings_config::Action::ZoomOut => todo!(),
         }
     }
 
